akka-actor-tests/src/test/java/akka/event/LoggingAdapterTest.java

```diff
@@ -128,7 +128,7 @@ protected Void match(Object event) {
                     LogEvent log = (LogEvent) event;
                     assertEquals(message, log.message());
                     assertEquals(level, log.level());
-                    assertEquals(mdc, log.mdc().toString());
+                    assertEquals(mdc, log.getMDC().toString());
```

Comment:
It is only used within this test, and only with MDCs of size one, so this is why it works. The other purpose was to get Eclipse JDT confused about the API to remind me that a Java version is still needed ;-)